packages/account/src/BaseSmartAccount.ts

```diff
@@ -248,53 +248,52 @@ export abstract class BaseSmartAccount implements IBaseSmartAccount {
     this.validateUserOp(userOp, requiredFields);
 
     let finalUserOp = userOp;
-    const skipBundlerCall = skipBundlerGasEstimation ?? true;
+    const skipBundlerCall = skipBundlerGasEstimation ?? false;
     // Override gas values in userOp if provided in overrides params
     if (overrides) {
       userOp = { ...userOp, ...overrides };
     }
 
     Logger.log("userOp in estimation", userOp);
 
-    if (skipBundlerCall) {
-      if (this.paymaster && this.paymaster instanceof BiconomyPaymaster && paymasterServiceData?.mode === PaymasterMode.SPONSORED) {
-        if (!userOp.maxFeePerGas && !userOp.maxPriorityFeePerGas) {
-          throw new Error("maxFeePerGas and maxPriorityFeePerGas are required for skipBundlerCall mode");
+    if (this.paymaster && this.paymaster instanceof BiconomyPaymaster && paymasterServiceData?.mode === PaymasterMode.SPONSORED) {
+      if (!userOp.maxFeePerGas && !userOp.maxPriorityFeePerGas) {
+        throw new Error("maxFeePerGas and maxPriorityFeePerGas are required for skipBundlerCall mode");
```

Comment:
If not available can't be make a call to biconomy_getGasFeeValues in parallel to the entire execution?